opal/mca/common/ucx/common_ucx_wpool.c

```diff
@@ -197,23 +250,24 @@ opal_common_ucx_wpool_init(opal_common_ucx_wpool_t *wpool,
     wpool->dflt_winfo = winfo;
     OBJ_RETAIN(wpool->dflt_winfo);
 
-    status = ucp_worker_get_address(wpool->dflt_winfo->worker,
-                                    &wpool->recv_waddr, &wpool->recv_waddr_len);
+    status = ucp_worker_get_address(wpool->dflt_worker,
```

Comment:
Is this a merge conflict with #7632? (I had changed `dflt_worker` to `dflt_winfo` so that the `winfo` can be locked in `opal_common_ucx_wpool_progress`)

Reply:
You are right - I have this fixed in the upcoming push, but we're working on a couple other issues before hand. Thanks Joseph